spec/models/miq_request_workflow_spec.rb

```diff
@@ -5,6 +5,14 @@
   let(:ems_folder) { FactoryBot.create(:ems_folder) }
   let(:datacenter) { FactoryBot.create(:ems_folder, :type => "Datacenter") }
 
+  before("#validate") do
```

Comment:
Can this `before` block live inside the `context` that it applies to for consistency with the rest of the code base?

Reply:
Ok, done.